doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_flpr.rst

```diff
@@ -0,0 +1,175 @@
+.. _ug_nrf54h20_flpr:
+
+Working with the FLPR core
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   The FLPR core support in the |NCS| is currently :ref:`experimental<software_maturity>`.
+
+The nRF54H20 SoC includes a dedicated VPR CPU, based on RISC-V architecture, known as the *fast lightweight peripheral processor* (FLPR).
+The FLPR core can be used to manage specific peripherals through the appropriate Zephyr Device Driver API:
+
+* ##List to be completed
+
+.. _vpr_flpr_nrf54h20_initiating:
+
+Using FLPR with Zephyr multithreaded mode
```

Comment:
I would rephrase the title. Right now it's ambiguous if FLPR or FLPR's owner runs multithreaded Zephyr. How about "Zephyr multithreaded mode on FLPR"? Or "Using Zephyr multithreaded mode on FLPR"?